packages/nextjs-template/components/DendronNotice.tsx

```diff
@@ -0,0 +1,59 @@
+import { ReactNode } from "react";
+import { useToggle } from "../hooks/useToggle";
+
+export const DendronNotice = ({
+  show = false,
+  children,
+}: {
+  show: boolean;
+  children: ReactNode;
+}) => {
+  const [val, toggle] = useToggle(show);
```

Comment:
can you change the component to be an [alert (info)](https://ant.design/components/alert/) for better visual contrast?